src/sentry/testutils/skips.py

```diff
@@ -67,11 +69,23 @@ def _requires_snuba() -> None:
 @pytest.fixture(scope="session")
 def _requires_kafka() -> None:
     kafka_conf = settings.SENTRY_DEVSERVICES["kafka"](settings, {})
+    zk_conf = settings.SENTRY_DEVSERVICES["zookeeper"](settings, {})
     (port,) = kafka_conf["ports"].values()
 
     if not _service_available("127.0.0.1", port):
         pytest.skip("requires kafka server running")
 
+    try:
+        check_health(
+            "kafka",
+            {
+                "kafka": kafka_conf,
+                "zookeeper": zk_conf,
+            },
+        )
+    except Exception as e:
```

Comment:
ideally we'd catch a more specific type than `Exception`

Reply:
Switched to subprocess.CalledProcessError